backup.sh

```diff
@@ -0,0 +1,2 @@
+python manage.py dbbackup
```

Comment:
This isn't going to work as is. When we run it from cron, there is no environment so we have to specify full paths. Here's a sample: 

```
#!/bin/bash

cd /home/sys-autograder/autograder
source bin/activate
cd AutoGrader
/home/sys-autograder/autograder/bin/python manage.py dbbackup 
```

Also, when you do the other PR, `requirements.txt` is going to be in conflict.

Reply:
Resolved conflict and updated backup.sh 

USAGE:
- Under Virtualenv
`backup.sh /home/sys-autograder/autograder virtualenv`

- Basic usage
`backup.sh /home/sys-autograder/autograder`

Updated file: https://github.com/BilalZaib/AutoGrader/pull/58/files